lib/models/live_stream_model.dart

```diff
@@ -0,0 +1,41 @@
+class LiveStreamModel {
+  LiveStreamModel({
+    required this.liveStreamId,
+    required this.title,
+    required this.startTime,
+    required this.viewers,
+    required this.channelId,
+    required this.directorId,
+    required this.directorName,
+  });
+
+  factory LiveStreamModel.fromSnapshot(/*DocumentSnapshot*/ dynamic snapshot) {
```

Comment:
Let's use `dynamic` for now—once we have Cloud Firestore up & running, we can start using the types from that package.